features/step_definitions/candidates/bookings/feedback_steps.rb

```diff
@@ -0,0 +1,8 @@
+Given "I have an attended booking" do
+  placement_request = FactoryBot.create(:placement_request, :with_attended_booking)
+  @booking = placement_request.booking
+end
+
+When "I visit the booking feedback link" do
+  visit new_candidates_booking_feedback_path(@booking.token)
```

Comment:
Oof, I'm going to slowly back away from that one and suggest we add some tests to that method 😅 I don't _think_ it would work here though as this also needs to be an absolute URL with the host? I went off how it was done elsewhere; its awkward as we're not in a view so need to provide the host manually

Reply:
That's fine by me